libpod/stats.go

```diff
@@ -33,13 +33,17 @@ func (c *Container) GetContainerStats(previousStats *ContainerStats) (*Container
 	if err != nil {
 		return nil, err
 	}
-
-	cgroup, err := cgroups.Load(cgroups.V1, cgroups.StaticPath(cgroupPath))
+	v1CGroups := GetV1CGroups(getExcludedCGroups())
+	if err != nil {
```

Comment:
No need for the error here, this doesn't return an error

Reply:
fixed